examples/istioctl-analyze/README.md

```diff
@@ -0,0 +1,34 @@
+# Istioctl Analyze
+
+The `istioctl-analyze` KRM config function detects potential issues with your
+Istio configuration. It takes in a List of KRM configs sourced from a local
+package and outputs validation errors by adding a `results` field to the List.
```

Comment:
The user doesn't care nitty gritty of the functions spec. Keep this high level regarding why the user needs to care.

Reply:
Made it more succinct